### PR TITLE
E2E tests for home page 

### PR DESCRIPTION
**Pull request by Marcelinna**
Add E2E test for home page

Add test for providers, dedicated for and platforms list.
Add test for scientific domain and categories boxes.

### CODE REVIEW

File: test/system/package.json [hunk 1]
```diff
@@ -4,9 +4,10 @@
   "description": "Integration tests",
   "main": "",
   "scripts": {
-    "cy:run": "TZ=Europe/Warsaw cypress run --browser chrome --headless --quiet",
+    "cy:run": "TZ=Europe/Warsaw cypress run --browser chrome --headless --quiet --spec 'cypress/integration/*.spec.ts'",
     "cy:run-debug": "DEBUG=cypress:* TZ=Europe/Warsaw cypress run --browser chrome --headless --quiet",
-    "cy:run-browser": "TZ=Europe/Warsaw cypress run --browser chrome --quiet"
+    "cy:run-browser": "TZ=Europe/Warsaw cypress run --browser chrome --quiet",
+    "cy:run-regression-tests": "TZ=Europe/Warsaw cypress run --browser chrome --headless --quiet --spec 'cypress/integration/regression_tests/*/*.spec.ts'"
```
Comment (danielkryska) [inline test/system/package.json:10]: Please attach new command of regression tests to the Github workflow (try one of below) in file `.github/workflows/e2e.yml`.

Replace `command: npm run cy:run --prefix ./test/system` with

```
command: |
    npm run cy:run --prefix ./test/system
    npm run cy:run-regression-tests --prefix ./test/system
```

or

```
command: npm run cy:run --prefix ./test/system && npm run cy:run-regression-tests --prefix ./test/system

File: test/system/cypress/integration/regression_tests/home-page/scientific-domain_categories.spec.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
+describe("Scientific domain", () => {
+  before(() => {
+    cy.visit("/");
+  });
+
+  it("should click on scientific domain image and go to service page with selected filter", () => {
+    cy.intercept("GET", "/services*").as("services");
+    cy.get("a[href*='services'][data-e2e='branch-link']").then((element) => {
+      for (let i = 0; i < element.length; i++) {
```
Comment (danielkryska) [inline test/system/cypress/integration/regression_tests/home-page/scientific-domain_categories.spec.ts:9]: Do we need to add additional complexity to testing? Won't be enough to select only first element to check behaviour of an app?

File: test/system/cypress/integration/regression_tests/home-page/scientific-domain_categories.spec.ts [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+      }
+    });
+  });
+  it("should click on categories image and go to services page with selected category", () => {
+    cy.intercept("GET", "/categories/*").as("categories");
+    cy.get("li")
+      .contains("Categories")
+      .click();
+    cy.get("a[href*='categories'][data-e2e='branch-link']").then((element) => {
+      for (let i = 0; i < element.length; i++) {
```
Comment (danielkryska) [inline test/system/cypress/integration/regression_tests/home-page/scientific-domain_categories.spec.ts:30]: Do we need to add additional complexity to testing? Won't be enough to select only the first element to check behavior of the app?

File: test/system/cypress/integration/regression_tests/home-page/scientific-domain_categories.spec.ts (hunk 2 above)
Comment (Marcelinna) [reply test/system/cypress/integration/regression_tests/home-page/scientific-domain_categories.spec.ts:9]: I changed for first element

File: test/system/cypress/integration/regression_tests/home-page/scientific-domain_categories.spec.ts (hunk 3 above)
Comment (Marcelinna) [reply test/system/cypress/integration/regression_tests/home-page/scientific-domain_categories.spec.ts:30]: I changed for first element

File: test/system/cypress/integration/regression_tests/home-page/scientific-domain_categories.spec.ts [hunk 4]
```diff
@@ -0,0 +1,29 @@
+describe("Scientific domain", () => {
+  beforeEach(() => {
+    cy.visit("/");
```
Comment (Marcelinna) [inline test/system/cypress/integration/regression_tests/home-page/scientific-domain_categories.spec.ts:3]: comment

**Review by danielkryska — APPROVED** [review]

**Review by kmarszalek — APPROVED** [review]

**Pull request merged**